vue.config.js

```diff
@@ -1,6 +1,13 @@
 // optional file, loaded automatically by @vue/cli-service if present next to package.json
 var webpack = require('webpack')
 
+/**
+ * GraphQL URL used to rewrite requests when running a devServer. This will be used by a proxy in the devServer.
+ * @type {string}
+ */
+const user = process.env.CYLC_USER || process.env.USER
```

Comment:
Before we were enforcing developers to use `CYLC_USER` and add it to their `.env.local` or similar [file](https://cli.vuejs.org/guide/mode-and-env.html#local-only-variables). I thought it would be simpler to default to use the process user if no value provided.

This way developers can be more lazy and avoid this step if they are running the project locally (which I think we all are at this point, with no remote dev deployments yet?)